ui/src/timeMachine/components/TimeMachine.scss

```diff
@@ -53,3 +53,14 @@
     left: 0;
   }
 }
+
+.time-machine--view__empty {
+  background-position: center center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-image: url('../../assets/images/radial-power-level.svg');
```

Comment:
can you change this to absolute paths with `'~src/assets/images/radial-power-level.svg'`?

Reply:
It throws an error when I try that